testsuite/MDAnalysisTests/analysis/test_gnm.py

```diff
@@ -0,0 +1,49 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 fileencoding=utf-8
+#
+# MDAnalysis --- http://www.MDAnalysis.org
+# Copyright (c) 2006-2015 Naveen Michaud-Agrawal, Elizabeth J. Denning, Oliver
+# Beckstein and contributors (see AUTHORS for the full list)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+from __future__ import print_function
+
+import MDAnalysis
+import MDAnalysis.analysis.gnm
+
+from numpy.testing import (TestCase, dec, assert_equal, assert_almost_equal)
+import numpy as np
+
+import tempdir
+
+from MDAnalysisTests.datafiles import PSF, DCD
+from MDAnalysisTests import parser_not_found
+
+
+class TestGNM(TestCase):
+    @dec.skipif(parser_not_found('DCD'),
+                'DCD parser not available. Are you using python 3?')
+    def setUp(self):
+        self.tmpdir = tempdir.TempDir()
+        self.universe = MDAnalysis.Universe(PSF, DCD)
```

Comment:
Would the test make sense on `MDAnalysisTests.datafiles.XTC` rather than DCD? It would allow to run the test on python 3.